main/acle.rst

```diff
@@ -5822,3 +5835,183 @@ Instructions
+|      svint32_t svset_neonq[_s32](svint32_t vec, int32x4_t subvec)              |
++--------------------------------------------------------------------------------+
+| .. code:: c                                                                    |
+|                                                                                |
+|      svint64_t svset_neonq[_s64](svint64_t vec, int64x2_t subvec)              |
++--------------------------------------------------------------------------------+
+| .. code:: c                                                                    |
+|                                                                                |
+|      svuint8_t svset_neonq[_u8](svuint8_t vec, uint8x16_t subvec)              |
++--------------------------------------------------------------------------------+
+| .. code:: c                                                                    |
+|                                                                                |
+|      svuint16_t svset_neonq[_u16](svuint16_t vec, uint16x8_t subvec)           |
++--------------------------------------------------------------------------------+
+| .. code:: c                                                                    |
+|                                                                                |
+|      svuint32_t svset_neonq[_u32](svuint32_t vec, uint32x4_t subvec)           |
++--------------------------------------------------------------------------------+
+| .. code:: c                                                                    |
+|                                                                                |
+|      svuint64_t svset_neonq[_u64](svuint64_t vec, uint64x2_t subvec)           |
++--------------------------------------------------------------------------------+
+| .. code:: c                                                                    |
+|                                                                                |
+|      svfloat16_t svset_neonq[_f16](svfloat16_t vec, float16x8_t subvec)        |
++--------------------------------------------------------------------------------+
+| .. code:: c                                                                    |
+|                                                                                |
+|      svfloat32_t svset_neonq[_f32](svfloat32_t vec, float32x4_t subvec)        |
++--------------------------------------------------------------------------------+
+| .. code:: c                                                                    |
+|                                                                                |
+|      svfloat64_t svset_neonq[_f64](svfloat64_t vec, float64x2_t subvec)        |
++--------------------------------------------------------------------------------+
+
+``svget_neonq``
+---------------
+
+These intrinsics return a NEON vector from the first 128 bit subvector of
```

Comment:
Same comments here as for `vset_neonq`:

* we need to define what happens for big-endian targets
* there should be a bf16 version too

Reply:
added bf16 version. big-endian isnt yet resolved.